logbook-core/src/test/java/org/zalando/logbook/ObfuscatedHttpRequestTest.java

```diff
@@ -45,6 +45,20 @@
             (contentType, body) -> body.replace("s3cr3t", "f4k3"));
 
     @Test
+    public void shouldNotFailOnInvalidUri() {
+        final String invalidUri = "/af.cgi?_browser_out=.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2Fetc%2Fpasswd";
```

Comment:
Yeah, seen in the wild by one of our services, along with a dozen similar
URIs, all leading 500s b/c of an IAE in URI.create.
Am 22.02.2016 12:32 schrieb "Willi Schönborn" <notifications@github.com>:

> In
> logbook-core/src/test/java/org/zalando/logbook/ObfuscatedHttpRequestTest.java
> <https://github.com/zalando/logbook/pull/49#discussion_r53613418>:
>
> > @@ -45,6 +45,20 @@
> >              (contentType, body) -> body.replace("s3cr3t", "f4k3"));
> >
> >      @Test
> > +    public void shouldNotFailOnInvalidUri() {
> > +        final String invalidUri = "/af.cgi?_browser_out=.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2F.|.%2Fetc%2Fpasswd";
>
> oO
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/logbook/pull/49/files#r53613418>.
>